doc/04-Upgrading.md

```diff
@@ -3,6 +3,54 @@
 Specific version upgrades are described below. Please note that version upgrades are incremental.
 If you are upgrading across multiple versions, make sure to follow the steps for each of them.
 
+## Upgrading to Icinga DB v1.1.2
+
+Please apply the `1.1.2.sql` upgrade script to your database. For package installations, you can find this file at
+`/usr/share/icingadb/schema/mysql/upgrades/` or `/usr/share/icingadb/schema/pgsql/upgrades/`, depending on your
+database vendor.
+
+!!! warning
+
+    In order to fix a loading performance issue of the `icingadb/history` view, this upgrade script adds an appropriate
```

Comment:
You might want to explicitly mention Icinga Web here. While I get that this is probably a relative URL in Icinga Web, I don't think it'll be clear to the average user. So I'd just refer to it as something like "history view in Icinga DB Web".